SUBDOMAIN_TAKEOVER/description.md

```diff
@@ -0,0 +1,4 @@
+#Subdomain takeover
+
+Subdomain takeover attacks are a class of security issues where an attacker is able to seize control of an
+organization's subdomain via cloud services like GitHub , AWS or Azure. They commonly happen when web projects are ended but the subdomain DNS entries are not fully shut down.
```

Comment:
It would be nice to state the different impacts of subdomain takeover. You can check https://www.honeybadger.io/blog/subdomain-takeover/ for inspiration.